tools/kubectl-karpenter/README.md

```diff
@@ -0,0 +1,28 @@
+# kubectl-karpenter
+
+kubectl-karpenter is a simple CLI tool to bump the API manifests from `v1alpha5` to `v1beta1`.
+It converts `Provisioner` to `NodePool` and `AWSNodeTemplate` to `EC2NodeClass`.
+
+## Installation 
+
+```
+go install github.com/aws/karpenter/tools/kubectl-karpenter/cmd/kubectl-karpenter
+```
+
+## Usage:
+
+```
+# Convert a single Provisioner file to NodePool
```

Comment:
Do each of these commands need a pipe character?

Reply:
No, can work even without. I've removed them